jvm-common/src/main/kotlin/org/digma/intellij/plugin/idea/navigation/NavigationDiscoveryChangeService.kt

```diff
@@ -299,6 +299,8 @@ class NavigationDiscoveryChangeService(private val project: Project, private val
 
                         delay(1000)
                     }
+                } catch (ce: CancellationException) {
```

Comment:
so throwing the exception will fix the error in posthod? throwing it is passing the responsibility of handling it to the caller, right?